src/Sql/Sql.Test/ScenarioTests/DatabaseBackupTests.cs

```diff
@@ -193,14 +193,14 @@ public void TestShortTermRetentionPolicy()
             }
         }
 
-        [Fact]
+        [Fact(Skip = "Location 'East US 2 EUAP' is not accepting creation of new Windows Azure SQL Database servers at this time.'")]
```

Comment:
Could you please try another location to run this test?

Reply:
I have tried west central us which should be close to euap regions and provisioning is not supported. 
`"Provisioning of database with zone backup redundancy is not supported in this location."`